bigtable/client/internal/unary_rpc_utils.h

```diff
@@ -155,12 +155,12 @@ struct CallWithRetry {
   static typename std::enable_if<
       CheckSignature<MemberFunction>::value,
       typename CheckSignature<MemberFunction>::ResponseType>::type
-  MakeCall(ClientType &client,
-           std::unique_ptr<bigtable::RPCRetryPolicy> rpc_policy,
-           std::unique_ptr<bigtable::RPCBackoffPolicy> backoff_policy,
-           MemberFunction function,
-           typename CheckSignature<MemberFunction>::RequestType const &request,
-           char const *error_message) {
+  CallWithRetry(
```

Comment:
can this be renamed to makeCall. It can be used to even call without retry by creating a RPCPolicy with NoRetry

Reply:
As discussed in #269, that would not be my preferred approach.